base_location_nuts/wizard/nuts_import.py

```diff
@@ -133,15 +132,16 @@ def _download_nuts(self, url_base=None, url_path=None, url_params=None):
         logger.info('Download successfully %d bytes' %
                     len(res_request.content))
         # Workaround XML: Remove all characters before <?xml
-        pattern = re.compile(r'^.*<\?xml', re.DOTALL)
-        content_fixed = re.sub(pattern, '<?xml', res_request.content)
-        if not re.match(r'<\?xml', content_fixed):
+        pattern = re.compile(r'^.*<\?xml'.encode('utf-8'), re.DOTALL)
+        content_fixed = re.sub(pattern, b'<?xml',
```

Comment:
It doesn't make much sense to manually compile a pattern that you're gonna use just once, even more when python already autocaches often-used pattern compilations.

Check explanation here: https://docs.python.org/3/library/re.html#re.compile